docs/Editing-Configuration-Files.md

```diff
@@ -101,6 +101,7 @@ Here is a sample of the three basic types: respectively Boolean, Number and Stri
  * **peer-limit-global:** Number (default = 200)
  * **peer-limit-per-torrent:** Number (default = 50)
  * **peer-socket-tos:** String (default = "default") Set the [Type-Of-Service (TOS)](https://en.wikipedia.org/wiki/Type_of_Service) parameter for outgoing TCP packets. Possible values are "default", "lowcost", "throughput", "lowdelay" and "reliability". The value "lowcost" is recommended if you're using a smart router, and shouldn't harm in any case.
+ * **client_reqq:** Number (default = 2000) The number of outstanding block requests a peer is allowed to queue in the client. The higher this number, the higher the max possible upload speed towards each peer.
```

Comment:
explain where 2000 comes from.  lt-rb, eg?

Reply:
I didn't do this because @ckerr specifically requested not to in #5400.